Sources/GRPCNIOTransportHTTP2Posix/Config+TLS.swift

```diff
@@ -15,6 +15,7 @@
  */
 
 private import GRPCCore
+public import NIO
```

Comment:
NIO is an umbrella import, you only need `EventLoopPromise` here so `NIOCore` is sufficient. 

```suggestion
public import NIOCore
```